lib/pkg/flow-base/lib/FlowProcess.js

```diff
@@ -342,7 +342,7 @@ export class FlowProcess {
       ...args,
       '--from', 'nuclide',
     ];
-    const pathToFlow = getPathToFlow();
+    const pathToFlow = await getPathToFlow();
```

Comment:
How much slower do you think this call is? I mainly worry about things like autocomplete, where adding 100ms can really degrade performance.